gridcapa-task-manager-app/src/main/java/com/farao_community/farao/gridcapa/task_manager/app/entities/ProcessFile.java

```diff
@@ -41,12 +45,15 @@ public class ProcessFile implements Comparable<ProcessFile> {
     @Column(name = "file_type")
     private String fileType;
 
+    @TimeZoneStorage(TimeZoneStorageType.NORMALIZE)
```

Comment:
I think it's better to define the timezone storage in the application.yml like this

```
spring:
  jpa:
    properties:
      hibernate:
        timezone:
          default_storage: NORMALIZE
  ```
  
And everything will work as before, instead of using annotation on each OffsetDateTime, and use the annotation if one day you have a specific case where you want to store an offset other than "Normalize"

Reply:
You're right, i pushed suggested modifications in gridcapa-task-manager & gridcapa-deployment